addons/missileguidance/XEH_postInit.sqf

```diff
@@ -24,18 +24,18 @@
 [DIK_NUMPAD2, [false, false, false]], false, 0] call CBA_fnc_addKeybind;  // Numpad 2
 
 ["ACE3 Weapons", QGVAR(mclosLeft), LLSTRING(mclosLeft), {
```

Comment:
```suggestion
["ACE3 Weapons", QGVAR(mclos_Left), LLSTRING(mclosLeft), {
```
this is clunky but i think nessacary

the first time a keybind is added, the default binding gets added to the user profile and will stay that way
even if the default bind changes

so the only way to fix is to change the naming-key
otherwise everyone who played with have backward keys

Reply:
well thats not ideal. since people are already saying how they rebound the controls to other keys, wouldn't this split their control scheme? I think that reversing direction is going to be more wanted than just losing control completely

Reply:
probably best to just rename all 4

Reply:
is there any way to add a keybind but not have it be rebindable? I would rather keep these bad names and have them mapped as expected, but then add the correct ones as well